app/livechat/server/api/v1/message.js

```diff
@@ -134,12 +135,12 @@ API.v1.addRoute('livechat/message/:_id', {
 			const result = Livechat.updateMessage({ guest, message: { _id: msg._id, msg: this.bodyParams.msg } });
 			if (result) {
 				const message = Messages.findOneById(_id);
-				return API.v1.success({ message });
+				return API.v1.success({ message: normalizeMessageAttachments(message) });
```

Comment:
Do you mind of checking the `message.file` before returning? I think we should always return the `message` instead of the `normalizeMessageAttachments`.